modules/network-observability-netobserv-cli-uninstalling.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+
+// * observability/network_observability/netobserv_cli/netobserv-cli-install.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="network-observability-cli-uninstall_{context}"]
+= Uninstalling the Network Observability CLI
+
+You can uninstall the CLI by running `oc netobserv cleanup`. This command removes all the CLI components.
```

Comment:
Running cleanup clears what's deployed on the cluster; not the command line interface installed on your local machine.

```suggestion
You can manually cleanup the CLI workload by running `oc netobserv cleanup`. This command removes all the CLI components from your cluster.

This command is automatically executed by the client when ending a capture but may be required in case of connectivity issues. 
```

Reply:
FYI - I have updated this uninstalling to be a "Cleaning" topic instead and relocated to "Using the CLI" section: https://75055--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/network_observability/netobserv_cli/netobserv-cli-using#network-observability-cli-uninstall_netobserv-cli-using